workflows/scRNAseq/pseudobulk-worflow-decoupler-edger/pseudo-bulk_edgeR-tests.yml

```
@@ -0,0 +1,54 @@
+- doc: Test outline for pseudo-bulk_edgeR
+  job:
+    Source AnnData file:
+      class: File
+      location: https://zenodo.org/records/13929549/files/Source%20AnnData%20file.h5ad?download=1
+      filetype: h5ad
+    contrasts_files:
+      class: File
+      path: test-data/contrasts_files.txt
+      filetype: txt
+    'Pseudo-bulk: Fields to merge': null
+    Groupy column: cell_type
+    Sample key column: individual
+    Name Your Raw Counts Layer: counts
+    Factor fields: disease
+    Gene symbol column: gene_symbol
+    Formula: '~ 0 + disease '
+  outputs:
+    'pseudobulk_count_matrix':
+        has_text_matching:
+          expression: "ACAP2\t9.0\t18.0\t20.0\t68.0\t106.0\t122.0\t14.0\t259.0\t279.0\t184.0\t612.0\t293.0\t297.0\t46.0\t1.0\t0.0\t1.0\t12.0\t229.0\t151.0\t141.0\t309.0\t299.0\t181.0\t2.0\t2.0\t28.0\t15.0\t54.0\t210.0\t1.0\t1.0\t1.0\t11.0"
+          expression: "ACER3\t4.0\t25.0\t21.0\t110.0\t82.0\t91.0\t22.0\t326.0\t297.0\t211.0\t1004.0\t574.0\t370.0\t108.0\t0.0\t0.0\t2.0\t2.0\t188.0\t113.0\t135.0\t322.0\t324.0\t159.0\t7.0\t7.0\t32.0\t5.0\t33.0\t89.0\t2.0\t2.0\t8.0\t48.0"
+    'Pseudobulk Plot':
+      element_test:
+        has_size: 40116
+        delta: 2000 
+    'Filtered by expression':
+      element_test:
+        has_size: 23490
+        delta: 2000 
+    'Report Results: HTML File':
+      element_test:
+        has_size: 531761
+        delta: 25000
+    'Tables: DEG ':
```

```suggestion
    'Tables: DEG':
```
This needs changing in the workflow output label as well.